providers/linkedin/schemas.json

```diff
@@ -0,0 +1,537 @@
+{
+    "modules": {
+        "root": {
+            "id": "root",
+            "path": "rest",
+            "objects": {
+                "adAnalytics": {
+                    "displayName": "Adanalytics",
```

Comment:
Display names are supposed to be human readable (e.g. "Ad Analytics", not "Adanalytics"), please fix all the display names in this file.